rsl_tools.py

```diff
@@ -0,0 +1,78 @@
+import requests
+import zipfile
+import shutil
+import os
+import json
+import subprocess
+from version import randomizer_commit, randomizer_version
+
+def download_randomizer():
+    zippath = 'randomizer.zip'
+
+    # Make sure an old zip isn't sitting around
+    if os.path.isfile(zippath):
+        os.remove(zippath)
+
+    # Download the zipped randomizer
+    r = requests.get(f'https://github.com/Roman971/OoT-Randomizer/archive/{randomizer_commit}.zip', stream=True)
+    with open(zippath, 'wb') as fp:
+        for chunk in r.iter_content():
+            fp.write(chunk)
+    
+    # Extract the zip file and add __init__.py
+    zf = zipfile.ZipFile(zippath)
```

Comment:
Should use
```
    with zipfile.ZipFile(zippath, 'r') as zf:
        zf.extractall('.')
```
otherwise `os.remove(zippath)` will fail on the first time you download the zip.

Reply:
Great point. This was working fine for me but this is the safer way of doing it